src/handlers/track_table.rs

```diff
@@ -65,6 +65,25 @@ pub fn handler(key: Key, app: &mut App) {
                     }
                     TrackTableContext::AlbumSearch => {}
                     TrackTableContext::PlaylistSearch => {}
+                    TrackTableContext::MadeForYou => {
+                        let (playlists, selected_playlist_index) =
+                            (&app.library.made_for_you_playlists, &app.made_for_you_index);
+
+                        if let Some(selected_playlist) = playlists.pages[0]
```

Comment:
Instead of reading the array at index 0, you should use the `get_results` method implemented on the `ScrollableResultPages` struct. The will by default fetch the page in the struct's index.